src/web/mjs/connectors/Comikey.mjs

```diff
@@ -0,0 +1,90 @@
+import Connector from '../engine/Connector.mjs';
+
+export default class ComiKey extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'comikey';
+        super.label = 'Comikey';
+        this.tags = ['manga', 'webtoon', 'english'];
+        this.url = 'https://comikey.com';
+        this.chapterUrl = 'https://relay-us.epub.rocks/consumer/COMIKEY/series/';
+    }
+
+    _getMangaListFromPages(page) {
+        page = page || 1;
+        let uri = new URL('/sapi/comics', this.url);
+        uri.searchParams.set('page', page);
+        let request = new Request(uri.href, this.requestOptions);
+        return this.fetchJSON(request)
+            .then(data => {
+                let mangaList = data.results.map(manga => {
+                    return {
+                        id: manga.id + ',' + manga.e4pid,
+                        title: manga.name
+                    };
+                });
+                if (mangaList.length == 25) {
+                    return this._getMangaListFromPages(page + 1)
+                        .then(mangas => mangaList.concat(mangas));
+                } else {
+                    return Promise.resolve(mangaList);
+                }
+            });
+    }
+
+    _getMangaList(callback) {
+        this._getMangaListFromPages()
+            .then(data => callback(null, data))
+            .catch(error => {
+                console.error(error, this);
+                callback(error, undefined);
+            });
+    }
+
+    _getChapterList(manga, callback) {
+        let request = new Request(this.chapterUrl + manga.id.split(',')[1] + '/content?clientid=dylMNK5a32of', this.requestOptions);
```

Comment:
Is the clientID assumed to be consistent, or is this this expected to change?
Is this ID a personal ID or more like an API key?

Reply:
Not sure about the origin. I tried multiple sessions across multiple browsers and i'm getting this same id. Probably same for all desktop clients i guess.